frontend/packages/dev-console/src/components/buildconfig/BuildConfigForm.tsx

```diff
@@ -44,7 +44,9 @@ const BuildConfigForm: React.FC<FormikProps<BuildConfigFormikValues> & {
 
   const isStale = watchedBuildConfig?.metadata?.resourceVersion !== values.resourceVersion;
 
-  const formEditor = <BuildConfigFormEditor namespace={namespace} />;
+  const [formReload, setFormReload] = React.useState<number>(0);
```

Comment:
Maybe it's better to put this in formik state only. That way you wouldn't need to pass this down as a prop to children components several levels down to `GitSection`.

Reply:
okay, added formReloadCount to context